kotsadm/web/src/components/PreflightResultPage.jsx

```diff
@@ -126,10 +133,16 @@ class PreflightResultPage extends Component {
       .then((res) => {
         if (res.status === 200) {
           this.props.data?.refetch();
+        } else {
+          this.setState({
+            errorMessage: `Encountered an error while trying to re-run preflight checks: ${res.error}`
+          });
         }
       })
       .catch((err) => {
-        console.log(err);
+        this.setState({
+          errorMessage: err ? `Encountered an error while trying to re-run preflight checks: ${err.message}` : "Something went wrong, please try again."
```

Comment:
same here, we should reset `errorMessage` to `""` at the beginning of the function